content/docs/command-reference/plots/modify.md

```diff
@@ -35,10 +35,10 @@ Property names are passed as [options](#options) to this command (prefixed with
 - `-t <name_or_path>, --template <name_or_path>` - set a default
   [plot template](/doc/command-reference/plots#plot-templates).
 
-- `-x <field>` - set a default field or column name (or number) from which the X
+- `-x <field>` - set a default field or column name (or index) from which the X
   axis data comes from.
 
-- `-y <field>` - set a default field or column name (or number) from which the Y
+- `-y <field>` - set a default field or column name (or index) from which the Y
```

Comment:
In this case "index" is nowhere else in the doc but for consistency we should also keep "number" here, probably.